torq.sh

```diff
@@ -91,12 +100,19 @@ print() {
  }
 
 debug() {
-  if [[ -z $(findproc "$1") ]]; then                             
-    sline=$(startline "$1")                                                                         # get start line for process
-    printf "$(date '+%H:%M:%S') | Executing...\n$sline -debug\n\n"
-    eval "$RLWRAP $sline -debug"                                                                    # append flag to start in debug mode
+  procno=$(awk '/,'$1',/{print NR}' "$CSVPATH")
+  host=$(getfield "$procno" "host")
+  if [[ $host == "localhost" || $host == `hostname -i`  ||
+        ${hostips[@]}  =~ $host || ${hostnames[@]} =~ $host ]]; then
```

Comment:
are `hostips` & `hostnames` defined at this point in the code? I see them defined in the `start()` function, but I think if are running `debug()` function it is instead of `start()` function so do they not need to be either (i) defined here too or (ii) defined globally outside of these functions?

Reply:
Yes, you are correct.  I should be able to define these globally to use in `start()`, `stop()` and `debug()`.  I'll make this change now.